### DISCUSSION TITLE
Setup custom sound

### DISCUSSION THREAD

**Discussion by holemanstristan** [MiguelRipoll23/homebridge-securitysystem#82] [Q&A]
Hi There, 
thanks a lot for this plugin, that looks exactly what i need.
i'm using hoobs and i'm trying to configure the custom sound but i'm not sure to understand what need to be done in order to play a custom sound ? where the sound need to be store on raspberry ? 

thanks a lot for your help.

**Comment by MiguelRipoll23** [comment]
Hello!

Use the Homebridge UI to configure the audio feature on the plugin's settings, first enable the `Play Audio` setting and then install ffmpeg using the command `sudo apt install ffmpeg`.

Expand the `Advanced Options` section and expand `Audio` as well, set your audio path to /home/`your_current_username`/sounds. Now create a directory called `sounds` in that path and follow the instructions that would be created after a Homebridge restart. A README file would be created there, all you have to do is add some MP3 files with the name that the instructions indicates for the plugin to find and play your custom sound.

**Reply by holemanstristan** [reply]
tanks a lot and this sound can be played automatically on the homepod ?

**Reply by holemanstristan** [reply]
I created the folder sounds in home/username but when i restart homebridge it create a folder en-us but it's empty.

**Reply by MiguelRipoll23** [reply]
In order to play sounds to an AirPlay speaker you need import the sounds that you can find in this repository to your iTunes library, then create an automation on the Home app with a “Play media” action using as triggers the switches this plugin creates. The play audio action requires a Apple Music subscription at the moment.

**Reply by MiguelRipoll23** [reply]
That behavior is correct, a folder for your current select language has been created. You should see a README outside that folder with additional instructions. If you connect a speaker to your Homebrisge device using either the headphone jack or USB the sound will then play.

**Comment by plgrimsley** [comment]
There is another way to create a siren sound if you have an appleTV 4 or later and a TV that you can either turn on via your home app (doesn't matter how, IR or via CEC or via Homebridge or Home Assistant).  You can create a scene in you home app to play audio, which can either be via your Apple Music or your iTunes/Apple Music library.  Trigger the scene with an automation when the siren itself is triggered.  Best to create another so when the siren is silenced you can quickly pause the music/alarm sound.
